tests/beta/queuing/test_beta_measure.py

```diff
@@ -48,8 +49,11 @@ def test_annotating_obs_return_type(self, stat_func, return_type, op):
             A = op(0)
             stat_func(A)
 
-        assert q.queue == [A]
-        assert q.get_info(A) == {"return_type": return_type}
+        assert q.queue[:-1] == [A]
+        meas_proc = q.queue[-1]
```

Comment:
Maybe also could add a line asserting that `q.queue[-1]` is the object/type we expect?

Reply:
Good catch, thank you! Updated this as part of your next point :)